spec/spec_helper.rb

```diff
@@ -2,6 +2,7 @@
 
 require 'pry'
 require 'active_record'
+require 'rspec/its'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.